coffeine/pipelines.py

```diff
@@ -249,13 +259,22 @@ def make_filter_bank_transformer(
     if vectorization_params is not None:
         vectorization_params_.update(**vectorization_params)
 
-    def _get_projector_vectorizer(projection, vectorization,  kernel=None):
+    def _get_projector_vectorizer(projection, vectorization,
+                                  alignment_steps=None,
+                                  kernel=None):
         out = list()
         for name in names:
-            steps = [
-                projection(**projection_params_),
-                vectorization(**vectorization_params_)
-            ]
+            if alignment_steps is None:
+                steps = [
+                    projection(**projection_params_),
+                    vectorization(**vectorization_params_)
+                ]
+            else:
+                steps = [
+                    projection(**projection_params_)
+                ] + alignment_steps + [
```

Comment:
Can we be sure that this not buggy? We need to make sure that for every column (‘name’) independent instances are instantiated. Otherwise you risk some stateful behavior.